prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 20 minutes
+Actual:  27 minutes
+"""
+COUNT_WORDS = {}
+
+sentence = input("Text: ")
+for i in sentence.split():
+    if i not in COUNT_WORDS:
+        COUNT_WORDS[i] = 1
+    else:
+        COUNT_WORDS[i] += 1
+
+list = []
```

Comment:
list is a python syntax, rename the list will be better

Reply:
done it